src/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ include( XrdXml )
 include( XrdPosix )
 include( XrdFfs )
 include( XrdPlugins )
+include( XrdSsi )
```

Comment:
Yes, we want SSI and http to be part of the core. People depend on it being packaged that way. Ceph is problematic because of the library dependencies. However, let's keep Ceph he way it is for now.